src/contacts/view/ContactView.ts

```diff
@@ -800,7 +800,7 @@ export class ContactView extends BaseTopLevelView implements TopLevelView<Contac
 	}
 
 	private deleteSelectedContacts(): Promise<void> {
-		return deleteContacts(this.getSelectedContacts())
+		return deleteContacts(this.getSelectedContacts(), () => this.contactViewModel.listModel.selectNone())
```

Comment:
that's exactly why I was opposed to the default argument

Reply:
Yes. Apparently this bug also exists for the search view separately. This is such a mess...